### ISSUE TITLE
[Erlang] 

### ISSUE THREAD

**Issue by eproxus** [sublimehq/Packages#2160]
- Sublime Version: 3.2.2, Build 3211 + Packages version a35f59170ae4ceb98b177a9f555437cd9810a31a
- OS Version: macOS 10.14.6

Try-catch statements one one line gets colored wrongly:

``` erl
-define(_get_stacktrace_(),
        try exit('$get_stacktrace') catch exit:'$get_stacktrace':Stacktrace -> Stacktrace end).
%                ^ This is colored like a variable
%                                   ^ catch and exit are not highlighted properly
```

cc @deathaxe

**Comment by deathaxe** [comment]
Fixed by #2157

**Comment by deathaxe** [comment]
@wbond This can be closed as well.
